front/README.md

```diff
@@ -87,25 +87,22 @@ The operational studies application enables capacity studies to be carried out o
 #### Folder's tree
 - **components/**
   * **Helpers/**
-  * **HomeContent/**
   * **ManageTrainSchedule/**
   * **Project/**
   * **Scenario/**
   * **SimulationResults/**
   * **Study/**
-  * **TimetableSelector/**
 - consts.ts
 - Home.tsx
 - **views/**
-  * HomeContent.js
   * ManageTrainSchedule.jsx
   * Project.js
   * Scenario.js
   * SimulationResults.tsx
   * Study.js
```

Comment:
not sure it is helpful to have this in a readme, hard to maintain and paraphrasing the code.

Reply:
It was a demand from some other frontdev but I agree with you. It would be necessary only to precise each file/folder.